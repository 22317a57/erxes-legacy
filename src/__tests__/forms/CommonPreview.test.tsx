import { mount, shallow } from 'enzyme';
import CommonPreview from 'modules/forms/components/step/preview/CommonPreview';
import React from 'react';

describe('CommonPreview component', () => {
  const defaultProps = {
    title: 'string'
  };

  test('renders shallow successfully', () => {
    shallow(<CommonPreview {...defaultProps} />);
  });
});
