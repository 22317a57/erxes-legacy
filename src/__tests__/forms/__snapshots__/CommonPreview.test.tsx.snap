// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CommonPreview component snapshot matches 1`] = `
<div
  className="sc-aewfc dsStAJ"
>
  <div
    className="sc-isBZXS htStnM"
  >
    <div
      className="sc-OxbzP jocSpk"
      style={
        Object {
          "backgroundColor": undefined,
        }
      }
    >
      <div>
        string
      </div>
    </div>
    <div
      className="sc-lnrBVv jitmGX"
    >
      <div
        className="sc-fvLVrH kjchyg"
      />
    </div>
  </div>
</div>
`;
